### PR TITLE
*: add parser and definition for all unsupported builtin funcs (part 3) 

### PR DESCRIPTION
**Pull request by XuHuaiyu**
This commit servers for the following builtin functions:
acos, asin, atan, atan2, cos, cot, degrees, exp, pi, radians, sin, tan, truncate

@coocood @shenli @hanfei1991 @zimulala @tiancaiamao PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: expression/builtin.go [hunk 1]
```diff
@@ -133,22 +133,35 @@ var funcs = map[string]functionClass{
...
+	ast.Degrees:  &degreesFunctionClass{baseFunctionClass{ast.Degrees, 1, 1}},
+	ast.Exp:      &expFunctionClass{baseFunctionClass{ast.Exp, 1, 1}},
+	ast.Floor:    &floorFunctionClass{baseFunctionClass{ast.Floor, 1, 1}},
+	ast.Ln:       &logFunctionClass{baseFunctionClass{ast.Log, 1, 1}},
+	ast.Log:      &logFunctionClass{baseFunctionClass{ast.Log, 1, 2}},
+	ast.Log2:     &log2FunctionClass{baseFunctionClass{ast.Log2, 1, 1}},
+	ast.Log10:    &log10FunctionClass{baseFunctionClass{ast.Log10, 1, 1}},
+	ast.PI:       &piFunctionClass{baseFunctionClass{ast.PI, 0, 0}},
+	ast.Pow:      &powFunctionClass{baseFunctionClass{ast.Pow, 2, 2}},
+	ast.Power:    &powFunctionClass{baseFunctionClass{ast.Pow, 2, 2}},
```
Comment (hanfei1991) [inline expression/builtin.go]: ast.Power

File: parser/parser_test.go [hunk 2]
```diff
@@ -491,7 +491,7 @@ func (s *testParserSuite) TestExpression(c *C) {
 func (s *testParserSuite) TestBuiltin(c *C) {
 	defer testleak.AfterTest(c)()
 	table := []testCase{
-		// for buildin functions
+		// for buildtin functions
```
Comment (hanfei1991) [inline parser/parser_test.go]: builtin

File: expression/builtin.go [hunk 3]
```diff
@@ -133,22 +133,35 @@ var funcs = map[string]functionClass{
...
+	ast.Ceil:     &ceilFunctionClass{baseFunctionClass{ast.Ceil, 1, 1}},
+	ast.Ceiling:  &ceilFunctionClass{baseFunctionClass{ast.Ceiling, 1, 1}},
+	ast.Conv:     &convFunctionClass{baseFunctionClass{ast.Conv, 3, 3}},
+	ast.Cos:      &cosFunctionClass{baseFunctionClass{ast.Cos, 1, 1}},
+	ast.Cot:      &cotFunctionClass{baseFunctionClass{ast.Cot, 1, 1}},
+	ast.CRC32:    &crc32FunctionClass{baseFunctionClass{ast.CRC32, 1, 1}},
+	ast.Degrees:  &degreesFunctionClass{baseFunctionClass{ast.Degrees, 1, 1}},
+	ast.Exp:      &expFunctionClass{baseFunctionClass{ast.Exp, 1, 1}},
+	ast.Floor:    &floorFunctionClass{baseFunctionClass{ast.Floor, 1, 1}},
+	ast.Ln:       &logFunctionClass{baseFunctionClass{ast.Log, 1, 1}},
```
Comment (coocood) [inline expression/builtin.go]: ast.Ln

**Comment by XuHuaiyu** [comment]
@coocood  @hanfei1991  PTAL

**Pull request merged**